content/en/tracing/trace_explorer/facets.md

```diff
@@ -3,25 +3,26 @@ title: Span Facets
 kind: documentation
 description: 'Trace Facets and Facet Panel'
 further_reading:
-    - link: 'tracing/trace_explorer/'
-      tag: 'Documentation'
-      text: 'Trace Explorer'
+- link: 'tracing/trace_explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the Trace Explorer'
 ---
 
 ## Overview
 
-Facets are user-defined tags and attributes from your spans. They are useful for both qualitative and quantitative data analysis. You can use facets in the Trace Explorer to:
+Facets are user-defined tags and attributes from your spans. They are useful for both [qualitative](#qualitative-facets) and [quantitative](#quantitative-facets-measures) data analysis. Facets allow you to manipulate spans in your [Trace Analytics monitors][3], and in APM queries that appear on [dashboards][4] and in [notebooks][5].
+
+The [Trace Explorer][6] includes out-of-the-box facets such as `Status` and `Service`. You can use facets in the Trace Explorer to:
 
 - [Search for and filter spans][1]
 - Perform trace analytics
+- Start troubleshooting once your spans are ingested
```

Comment:
Now that users don't need facets to do these things, what is the use case for facets? CC: @kayayarai 

```suggestion
The [Trace Explorer][6] includes out-of-the-box facets such as `Status` and `Service`. You can use facets in the Trace Explorer to:

- [Search for and filter spans][1]
- Perform trace analytics
- Start troubleshooting once your spans are ingested
```